### ISSUE TITLE
Create Lesson route

### ISSUE THREAD

**Issue by ShaunLloyd** [bridge-school/content-dashboard#6]
### Value Statement
As a user, I want to be able to see a lesson that I previously saved.

### Acceptance Criteria
- [ ] Should have the same module bar without the list of modules on the side
- [ ] Module bar should be read-only (We'll worry about editing later)
- [ ] below the module bar, should be a list of each module header with links out the markdown content

**Comment by msukmanowsky** [comment]
@ShaunLloyd is this what I'm working on in here https://github.com/bridge-school/content-dashboard/tree/feature/cohort-listings? It sounds similar but I'm honestly not sure.

![apr-22-2018 22-56-10](https://user-images.githubusercontent.com/362755/39192573-759bce56-47a7-11e8-9189-3ef730d1c941.gif)

**Comment by ShaunLloyd** [comment]
@msukmanowsky Not quite! this is for the page that you would direct too after clicking into one of those list items
